adafruit_displayio_layout/layouts/grid_layout.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, x, y, width, height, grid_size, cell_padding):
         self.grid_size = grid_size
         self.cell_padding = cell_padding
         self._cell_content_list = []
+        self._cell_dictionary = {}
 
```

Comment:
The `grid_position` is already stored in _cell_content_list. Maybe we can use that instead of adding a _cell_dictionary.

Reply:
Ah yes we can look it up from the list. Good idea, I will change it to work that way.